cognite/neat/rules/models/_base.py

```diff
@@ -107,6 +107,18 @@ def from_list(cls, entity_strings: list[str], base_prefix: str | None = None, **
         ]
 
 
+class ParentClass(Entity):
+    type_: EntityTypes = EntityTypes.class_
```

Comment:
If I understand this correctly, then this should never be set on instantiation of the class, but be a fixed value for all instances. 
```suggestion
    type_: ClassVar[EntityTypes] = EntityTypes.class_
```
Remember to import `from typing import ClassVar`